homeassistant/components/reolink/config_flow.py

```diff
@@ -328,6 +330,9 @@ async def async_step_user(
                 {
                     vol.Optional(CONF_PORT): cv.positive_int,
                     vol.Required(CONF_USE_HTTPS, default=False): bool,
+                    vol.Required(
+                        CONF_BC_PORT, default=DEFAULT_BC_PORT
+                    ): cv.positive_int,
```

Comment:
Why aren't you using `cv.port`?

Reply:
Because I did not know cv.port existed, changed it now.
Thanks!